bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -434,8 +438,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
 
     Frame label="Energy Meters" icon="energy"
     {
-          Default item=iCENTRAL_Ta label="General"      icon="energy" valuecolor=[>3000="red"]
-          Default item=iCENTRAL_Tb label="Ground Floor" icon="energy" valuecolor=[>3000="red"]
+          Default item=iCENTRAL_Ta      label="General"                        icon="energy" valuecolor=[>3000="red"]
+          Default item=iCENTRAL_Tb      label="Ground Floor"                   icon="energy" valuecolor=[>3000="red"]
+          Default item=CENTRAL_Ta_day   label="General Energy Today"           icon="energy" valuecolor=[>3000="red"]
+          Default item=CENTRAL_Tb_day   label="Ground Floor Energy Today"      icon="energy" valuecolor=[>3000="red"]
+          Default item=CENTRAL_Ta_month label="General Energy This Month"      icon="energy" valuecolor=[>3000="red"]
+          Default item=CENTRAL_Tb_month label="Ground Floor Energy This Month" icon="energy" valuecolor=[>3000="red"]
```

Comment:
Does it make sense to use the same trigger value for color for today and month ?

Reply:
we could have month items in yellow   and day items in blue for instance. not a problem. I will change it

Reply:
The syntax is of course good, I was just asking myself if it makes sense in practice to use the same trigger value (3000) for day and month.
It is just an example so let's go with that.